R/object_usage_linter.R

```diff
@@ -165,8 +165,7 @@ extract_glued_symbols <- function(expr) {
       parse(text = xml2::xml_text(cl)),
       error = function(...) NULL,
       warning = function(...) NULL
-    )[[1L]]
-    if (is.null(parsed_cl)) next
```

Comment:
I can't think of how we could end up with multiple expressions (`[[1]]`) or a parsing error here... if there's a flaw in our thinking that might lead to that happening, we should let it cause an error & users can report it so we can tighten the logic elsewhere.